Modules/Module_Processing/xASL_init_DefaultEffectiveResolution.m

```diff
@@ -48,6 +48,17 @@
     return;
 end
 
+% Obtain the x.Q.MRAcquisitionType field from the json
```

Comment:
Are you sure we also have PulseSequence here? Because remember we concluded they should always be together (one is 2D/3D, the other is EPI/GRASE/spiral).